pearcej/cpp4python
# Move reading questions in the glossary to the summary page and delete check yourself section - chapter 8 (cpp4python-v2)

# Description

I moved the reading question that was on the glossary page and put it on the summary page.
## Related Issue



fix #182
## How Has This Been Tested?
1. Make changes locally.
2. Verify the changes.
![image](https://github.com/pearcej/cpp4python/assets/117699634/cdcd7dbb-9921-4fb1-bead-de3a485f732d)
